### PR TITLE
Fix Libxml2 (change dir and CRuntime)

### PR DESCRIPTION
**Pull request by xsacha**
Fix dir change

### CODE REVIEW

**Comment by xsacha** [comment]
Thanks. Updated.

File: cmake/projects/libxml2/hunter.cmake [hunk 1]
```diff
@@ -35,9 +36,15 @@ hunter_add_version(
...
 hunter_configuration_types(libxml2 CONFIGURATION_TYPES Release)
 if (MSVC)
+    hunter_check_toolchain_definition(NAME "_DLL" DEFINED _hunter_vs_md)
     hunter_pick_scheme(DEFAULT url_sha1_libxml2_msvc)
     # pkgconfig file not applicable
     set(_libxml_unrelocatable_text_files "")
+    hunter_cmake_args(
+        libxml2
+            CMAKE_ARGS
+                BOOST_BUILD_DYNAMIC_VSRUNTIME=${_hunter_vs_md}
```
Comment (ruslo) [inline cmake/projects/libxml2/hunter.cmake]: I guess name should be `LIBXML2_BUILD_DYNAMIC_VSRUNTIME`

Comment (xsacha) [reply cmake/projects/libxml2/hunter.cmake]: Fixed. Thanks.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3226
* https://travis-ci.org/ingenue/hunter/builds/327576747?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3229
* https://travis-ci.org/ingenue/hunter/builds/327627017?utm_source=github_status&utm_medium=notification

@xsacha Note that there is no real Windows toolchain are tested/uploaded now. Please send pull request with toolchains you want to see to be tested. See https://docs.hunter.sh/en/latest/creating-new/update.html#add-toolchains

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.227

**Pull request merged**